client/src/components/App.js

```diff
@@ -22,20 +22,20 @@ class App extends Component {
     ])
       .then(res => Promise.all(res.map(dataset => dataset.json())))
       .then(data => {
-        console.log(data, "DATA");
         this.setState({ users: data[0], matches: data[1], teams: data[2] });
+        console.log(this.state.matches);
```

Comment:
there shouldnt be any console logs in the frontend

Reply:
done.